hoomd/hpmc/compute.py

```diff
@@ -311,16 +311,32 @@ def sdf(self):
         self._cpp_obj.compute(self._simulation.timestep)
         return self._cpp_obj.sdf
```

Comment:
I can't comment further up. In the class level docstring, document the expansion histogram and how `\beta P` relates to both the compression and expansion sdfs.